### ISSUE TITLE
Player status (Alive, Dead, Spectator....)

### ISSUE THREAD

**Issue by Lewisparkle** [ShaneBeeStudios/HungerGames#57]
**Is your feature request related to a problem? Please describe:**
No.

**Describe the solution you'd like:**
Hi! Can you please add in the new version placerholders with status about the player? For example: Alive, Dead, Spectator.... It would be nice to add this in the tablist or scoreboard. Of course they would be translationable. Thanks!

**Describe alternatives you've considered:**
None.

**Comment by ShaneBeee** [comment]
Im a little confused, are you meaning the PAPI placeholders? or for the scoreboard in the game?

Gonna need more details

**Comment by Lewisparkle** [comment]
Yes, I mean PAPI placeholders. They should be able to add them even to the scoreboard, shouldn't they?
